src/video_core/control/channel_state_cache.cpp

```diff
@@ -0,0 +1,11 @@
+#include "video_core/control/channel_state_cache.inc"
```

Comment:
Missing copyright header

```suggestion
// SPDX-FileCopyrightText: 2022 yuzu Emulator Project
// SPDX-License-Identifier: GPL-3.0-or-later

#include "video_core/control/channel_state_cache.inc"
```